specs/file_hierarchy_for_the_verification_of_os_artifacts.md

```diff
@@ -0,0 +1,568 @@
+- By default, VOA suggests that each verifier is stored in an individual file (that ideally reflects a unique identifier of that verifier in its filename).
+  However, in some technologies, separating verifiers into individual files may be uncommon or unreasonable.
+  In these technologies, bundles of verifiers may be stored in a shared file (this can apply equally to _artifact verifiers_ and _trust anchors_).
+- As an edge case, in some technologies, _artifact verifiers_ may be extremely short lived, so it may not be reasonable to store them in VOA.
+  In these cases, only a set of _trust anchors_ may be stored in VOA, while the _artifact verifiers_ are distributed with the signed artifacts.
+
+## File Hierarchy
+
+The _File Hierarchy for the Verification of OS Artifacts (VOA)_ is maintained as a directory structure on a system.
+It contains cryptographic public-key material for one or more technologies.
+
+The VOA hierarchy organizes _signature verifiers_ by [os], [purpose], [context] and [technology], and stores them in a directory structure of the form `$os/$purpose/$context/$technology/`.
+
+Vendors provide a set of _signature verifier_ files for their OS, in the VOA hierarchy format.
+
+Users of a VOA hierarchy (such as package installation software) can pick the relevant set of _signature verifiers_ for the verification of a specific artifact based on their location in the hierarchy.
+
+### Load paths
+
+VOA defines a list of _load paths_ with descending priority for system mode and user mode.
+
+_Signature verifiers_ are retrieved from hierarchies in these directories to validate artifacts based on specific [load logic].
+This is referred to as _verifier lookup_.
+
+The existence of multiple load paths allows system administrators and users to define custom sets of _signature verifiers_ for their system and applications.
+
+#### System mode
+
+The priority of load paths in system mode follows the definition of ["Storage Directories and Overrides" in the Configuration Files Sepcification], which is also reflected in the following list of paths:
+
+- `/etc/voa/`
+- `/run/voa/`
+- `/usr/local/share/voa/`
+- `/usr/share/voa/`
+
+#### User mode
+
+The priority of load paths in user mode mirrors that of the system mode.
+Refer to the [XDG Base Directory Specification] for details on default values.
+
+- `$XDG_CONFIG_HOME/voa/`
+- the `./voa/` directory in each directory defined in `$XDG_CONFIG_DIRS`
+- `$XDG_RUNTIME_DIR/voa/`
+- `$XDG_DATA_HOME/voa/`
+- the `./voa/` directory in each directory defined in `$XDG_DATA_DIRS`
+
+### Symlinking
+
+Symlinks can be used in the VOA hierarchy to point to files or directories below one of the [load paths].
+Symlinking to files or directories external to the load paths is not supported.
```

Comment:
why are symlinks rejected?  should an evaluator that discovers a symlink provide a warning explaining the rejection?